src/style.css

```diff
@@ -1 +1,52 @@
 /* add styles here */
+html {
+  margin: 0;
+  padding: 0;
+}
+
+body {
+  margin: 0;
+  padding: 0;
+}
```

Comment:
Don't use tag names for styling (except `html` and `body`). It's already mentioned in the first comment, but to emphasize, the `html` and `body` tags can share the same styling rule when they have identical styles.